doc/command_reference/command_reference.md

```diff
@@ -41,6 +41,82 @@ once the full list of updates has been read (end of list is indicated by a semic
 significantly more efficient and should always be the preferred option when the client wants to apply
 multiple updates that are known at the same time.
 
+## `modify` command
+
+The `modify` command
+
+```
+modify <relation> <key> <- <record>
+```
+
+modifies the value associated with primary key `<key>` in `<relation>`.  It is
```

Comment:
this is perhaps interesting, but I find this API asymmetric: people will expect that the same thing happens on outputs, You should indicate that this is not the case.